services/src/messaging/src/main/java/org/openkilda/messaging/model/FlowDto.java

```diff
@@ -59,7 +58,7 @@
     @JsonProperty("ignore_bandwidth")
     private boolean ignoreBandwidth;
 
-    @JsonProperty("periodic-pings")
+    @JsonProperty("periodic_pings")
```

Comment:
will this affect existing API for v1?

Reply:
thanks, good catch

Reply:
This should not affect API v1 because FlowPayload is used for this API. FlowDto is used for messaging.